packages/tools/examples/circleROIStartEndThresholdWithSegmentation/index.ts

```diff
@@ -236,8 +235,12 @@ async function run() {
 
   // Segmentation Tools
   toolGroup.addTool(CircleROIStartEndThresholdTool.toolName, {
-    calculatePointsInsideVolume: true,
+    /* Define if the stats are calculated while drawing the annotation or at the end */
+    computeStatsDuringEditing: true,
```

Comment:
can we make this not a breaking change, seems like you are removing a public configuraiton

Reply:
In fact we figured that calculatePointsInsideVolume was useless, as it was equivalent to showTextBox, if you need to display stats you need to calculatePointInsideVolume.